### ISSUE TITLE
How to add the color texturein blender?

### ISSUE THREAD

**Issue by merecesarchviz** [ReshotAI/gaussian-splatting-blender-addon#4]
hi,

First thank you for the addon, I was trying it, but after import the gaussian splatting file into blender, only get the points cloud no color or texture information cames! how i can add it the textures to the splatting?

thks

**Comment by merecesarchviz** [comment]
> The color comes when you enter Rendered View or Render the Result.
> 
> if you check the Option 'As point cloud (faster)' the Splats can only be rendered with Cycles. If you dont check the Option you can Render in Eevee and Cycles

and where is that options! here dont show me nothing that options

https://prnt.sc/PdOwXSHtwbzk

**Comment by merecesarchviz** [comment]
> ![image](https://user-images.githubusercontent.com/148103744/275647056-80f47545-0a5c-4e6b-b068-fed3fa4aea11.png)
> 
> After importing the GSpat set viewport shading to Rendered View (Red Arrow) In the Properties-Tab (Greed Box) go to Render Propertys and select render engine Cycles or Eevee (Blue Box)

I did al nothing appear in EEvEE or Cycles!! is all empty! only the point cloud is visible in the normal shading without texture or color!

**Comment by merecesarchviz** [comment]
> ![image](https://user-images.githubusercontent.com/148103744/275647056-80f47545-0a5c-4e6b-b068-fed3fa4aea11.png)
> 
> After importing the GSpat set viewport shading to Rendered View (Red Arrow) In the Properties-Tab (Greed Box) go to Render Propertys and select render engine Cycles or Eevee (Blue Box)

Here is what happens!: http://prntscr.com/FEUs6S2uzVNY

**Comment by scriptedAI-Git** [comment]
Ok i have no idea what went wrong with your import. The way you took looks fine.

1.) Can you check the Modifier tab? 
(Select Pointcloud -> Properties -> Modifier-Tab (Green Box in my screenshot))
The point cloud object should have a GeometryNodeModifier with a GaussianSplatting Node Tree.

2.) What Blender-Version do you use?

3.) Have you checked the Castle with the default SIBR_Viewer? Maybe something went wrong with the Splatting itself...
https://github.com/graphdeco-inria/gaussian-splatting#pre-built-windows-binaries

![image](https://github.com/ReshotAI/gaussian-splatting-blender-addon/assets/148103744/20d2b310-f43e-4228-9572-df37a6eac5aa)

**Comment by merecesarchviz** [comment]
> Ok i have no idea what went wrong with your import. The way you took looks fine. But somehow the 3D Gaussian Splatting Planel from you is empty (Red Arrow in my screenshot)
> 
> 1.) Can you check the Modifier tab? (Select Pointcloud -> Properties -> Modifier-Tab (Green Box in my screenshot)) The point cloud object should have a GeometryNodeModifier with a GaussianSplatting Node Tree.
> 
> 2.) What Blender-Version do you use?
> 
> 3.) Have you checked the Castle with the default SIBR_Viewer? Maybe something went wrong with the Splatting itself... https://github.com/graphdeco-inria/gaussian-splatting#pre-built-windows-binaries
> 
> ![image](https://user-images.githubusercontent.com/148103744/275758748-20d2b310-f43e-4228-9572-df37a6eac5aa.png)

Yes I open in the gaussian splatting viewer all really good! I create it also with that.

but something is wrong with the addon because checking your picture i dont have that options the the addon tab as i show you previous in my printscreen! Im using blender 3.6


Maybe dont work with this version because i only have the button option to " import gaussian splatting"

**Comment by merecesarchviz** [comment]
> Ok i have no idea what went wrong with your import. The way you took looks fine. But somehow the 3D Gaussian Splatting Planel from you is empty (Red Arrow in my screenshot)
> 
> 1.) Can you check the Modifier tab? (Select Pointcloud -> Properties -> Modifier-Tab (Green Box in my screenshot)) The point cloud object should have a GeometryNodeModifier with a GaussianSplatting Node Tree.
> 
> 2.) What Blender-Version do you use?
> 
> 3.) Have you checked the Castle with the default SIBR_Viewer? Maybe something went wrong with the Splatting itself... https://github.com/graphdeco-inria/gaussian-splatting#pre-built-windows-binaries
> 
> ![image](https://user-images.githubusercontent.com/148103744/275758748-20d2b310-f43e-4228-9572-df37a6eac5aa.png)

I also try in version 3.5 of blender and the same no options in the 3d Gaussian Splatting tab!! only the option to import

 ![image](http://prntscr.com/8cejhC-Gv1gp)

**Comment by scriptedAI-Git** [comment]
This is correct because if no GSplat is imported my screen loos the same
![image](https://github.com/ReshotAI/gaussian-splatting-blender-addon/assets/148103744/ab662214-21ec-4f2d-a044-aa51f096781d)

After i imported a GSplat the window changes
![image](https://github.com/ReshotAI/gaussian-splatting-blender-addon/assets/148103744/85a4d624-71bd-4904-8e1a-b995b6ee74a6)

Can you maybe make a smaller GSplat (small room for example) and try it with this?
Maybe you hit a limit in Blender =| 
I use Blender 3.6.0 by the way

**Comment by merecesarchviz** [comment]
You are true in version 3.5 after the import shows the options in the addon tab! but i still have the same issue no color or textures on the splatts, when i put in render shader mode all blank no color information on the splatts!

https://ibb.co/cyNj9PH

> This is correct because if no GSplat is imported my screen looks the same ![image](https://user-images.githubusercontent.com/148103744/275768182-ab662214-21ec-4f2d-a044-aa51f096781d.png)
> 
> After i imported a GSplat the window changes ![image](https://user-images.githubusercontent.com/148103744/275769212-85a4d624-71bd-4904-8e1a-b995b6ee74a6.png)
> 
> Can you maybe make a smaller GSplat (small room for example) and try it with this? Maybe you hit a limit in Blender =| I use Blender 3.6.0 by the way

You are true in version 3.5 after the import shows the options in the addon tab! but i still have the same issue no color or textures on the splatts, when i put in render shader mode all blank no color information on the splatts!

![image](https://i.ibb.co/4FjHLcG/bb.jpg)

**Comment by scriptedAI-Git** [comment]
Sorry to say but iam out of ideas now :(

**Comment by Fanie-Visagie** [comment]
Good day.

I may be of assistance...FIrst I just want to understand where you get the .splat filetype from ?? Standard output is ply ?? Using Blender 4.0...which is sick btw..

**Comment by LTKMN** [comment]
for what it's worth, I just went through this whole thing myself and went back in blender-time instead of forward.

I had started on 3.6.2 and then tried 3.2 and 2.9 with no success (it imported just plain dots, the menu didn't change or give export options, render view did nothing, same as above folks) and the solution for me was downloading 4.0.2 and then everything worked.

so maybe just update!

**Comment by Wladefant** [comment]
yeah the new version only works in 4.0 you can download an older version. These work in other versions but not in 4.0

**Comment by shinandrew** [comment]
Ive installed the add-on and have enabled it, but even the 3DGS menu in the screenshot above, where it says import Gaussian splatting, doesnt appear. Im using 4.0.2 and have also tried 4.0.0 Is there a way to manually open that menu or could something have gone wrong?
